packages/code/src/main.rs

```diff
@@ -94,13 +94,113 @@ fn transform_code(to: &str) {
                 print!("[{}]", json!({"name": "raw", "data": html}));
             }
         }
+        "latex" => {
+            if font_size != 12{
+                eprintln!("Font size is not supported in LaTeX");
+            }
+            print!("{}", highlight_latex(code, lang, theme, tab_size));
+        }
         other => {
             eprintln!("Cannot convert code to {other}");
         }
     }
 }
 
-fn get_style(
+fn highlight_latex(code: &str, lang: &String, tm: &str, tab_size: u64) -> String {
+    let ss = SyntaxSet::load_defaults_newlines();
+    let ts = ThemeSet::load_defaults();
+    let theme = match tm {
+        "ocean_dark" => &ts.themes["base16-ocean.dark"],
+        "ocean_light" => &ts.themes["base16-ocean.light"],
+        "mocha" => &ts.themes["base16-mocha.dark"],
+        "eighties" => &ts.themes["base16-eighties.dark"],
+        "github" => &ts.themes["InspiredGitHub"],
+        "solar_dark" => &ts.themes["Solarized (dark)"],
+        "solar_light" => &ts.themes["Solarized (light)"],
+        _ => &ts.themes["InspiredGitHub"],
+    };
+
+    if let Some(syntax) = ss.find_syntax_by_token(lang) {
+        let mut h = HighlightLines::new(syntax, theme);
+        let mut result = String::new();
+        let background_color = theme.settings.background.unwrap();
+        let r = background_color.r;
+        let g = background_color.g;
+        let b = background_color.b;
+        result.push('[');
```

Comment:
I think it may be error prone to construct arrays manually, maybe use `Vec<JsonEntry>` or `Vec<String>` or something and let `serde_json` serialize it?